app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md

```diff
@@ -0,0 +1,321 @@
+backlink: "../../"
+keywords:
+  - International
+  - Overseas
+  - Foreign
+  - Nationality
+  - Residency
+  - EEA
+  - Canada
+  - New Zealand
+  - Switzerland
+  - USA
+  - Visa
+  - Skilled worker visa
+  - EU
+  - EU Settlement Scheme
+  - NARIC
+  - UK NARIC
+  - ENIC
+  - UK ENIC
+  - European Network of Information Centre
+  - National Recognition Information Centre for the United Kingdom
+  - PhD
+  - Immigration
+  - Ireland
+  - Irish
+  - Common Travel Area
+  - Student Visa
+  - Home Office
+  - Graduate Visa
+  - Youth Mobility Scheme
+  - Japan
+  - Monaco
+  - Hong Kong
+  - Republic of Korea
+  - Taiwan
+  - Family Visa
+  - UK Ancestry Visa
+  - Abroad
+  - Europe
+  - European
+  - Settlement Scheme
+---
+
+If you’re applying for teacher training as a non-UK citizen, you may be able to get some financial help from the UK government. The information on this page is for:
+
+
+* non-UK citizens living in the UK
+* non-UK citizens living outside the UK
+* Irish citizens
+* refugees or asylum seekers
+
+## Teacher training fees
+
+As a non-UK candidate, you can expect to pay between £9,250 and about £32,000 to train to teach in England – fees vary between training providers. You’ll also have to cover your living costs.
+
+Depending on your immigration status, you may get a reduction in your fees to the rate paid by UK applicants (set at a maximum of £9,250). If you are eligible, this fee will be the same for all courses offered by accredited training providers. Please note – most non-UK candidates will not be eligible for this type of financial support. 
+
+If you have achieved highly in your bachelor’s degree and have at least 3 years’ relevant work experience, you can apply for a salaried teacher training course. On this type of course, you will not pay a fee, and you can earn money while you train in a school. However, these courses are limited in number and very competitive. 
+
+[Find teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses). 
+
+
+## Financial support for non-UK trainee teachers of languages and physics
+
+There are 3 types of financial support you can get if you’re interested in training to teach languages and physics:
+
+
+* an international relocation payment worth £10,000 for languages and physics
+* bursaries worth up to £27,000 for languages and physics
+* scholarships worth up to £29,000 for French, German, Spanish and physics
+
+
+If you are eligible, you can receive both an international relocation payment and a bursary or scholarship. You cannot receive both a bursary and a scholarship.
+
+### Get an international relocation payment (IRP) worth £10,000
+
+The IRP is designed to help pay the relocation expenses of languages or physics teacher trainees moving to the UK. 
+
+If you are eligible, you’ll receive a cash sum of £10,000 from DfE. You will not have to pay this money back and it can be spent in any way you wish. You can only receive the IRP once.
```

Comment:
```suggestion
If you are eligible, you’ll receive a cash sum of £10,000 from the Department for Education. You will not have to pay this money back and it can be spent in any way you wish. You can only receive the IRP once.
```